src/index.html

```diff
@@ -4,9 +4,474 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header body__header">
+      <div class="header__container">
+        <a href="index.html">
+          <img
+            class="header__logo"
+            src="./images/logo.png" alt="logo"
+            >
```

Comment:
```suggestion
          <img
              class="header__logo"
              src="./images/logo.png" alt="logo"
            >
```

Check please formatting in all places